js/main.js

```diff
@@ -0,0 +1,25 @@
+function toggleNavigationMenu() {
+  const ul = document.querySelector('#nav-menu');
+  ul.classList.toggle('mobile-nav-menu');
+}
+
+window.addEventListener('load', () => {
+  /* // Support For Hash Navigation
+  let $hash = window.location.hash;
+  const observer = new MutationObserver(() => {
+    if ($hash !== window.location.hash) {
+      $hash = window.location.hash;
+      console.log(window.location.hash);
+      // if(hash=="")
+    }
+  });
+  observer.observe(document, { subtree: true, childList: true }); */
```

Comment:
{OPTIONAL]
- Kindly remove the commented code. I'm sure you'd agree that a clean codebase is really nice to have, right? 😉